target_tools/logger.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Output logger"""
+import logging
+
+LOG = logging.getLogger()
+
+
+def get_logger(logger=None):
+    """Returns standard root logger for now"""
+    print(logger)
```

Comment:
@5amfung - both of these functions where print() is used are just dummy functions right now.  But the linter complains about unused vars, so that's why I just print them at this point.